hugegraph-server/hugegraph-example/src/main/java/org/apache/hugegraph/example/PerfExample2.java

```diff
@@ -34,6 +36,8 @@
  */
 public class PerfExample2 extends PerfExampleBase {
 
+    private static final Logger LOG = Log.logger(PerfExample2.class);
```

Comment:
Same issue as PerfExample1 - logger is declared but never used. Remove the unused import and logger declaration.